ui/pages/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -461,6 +493,7 @@ export default class ConfirmTransactionBase extends Component {
           networkName={networkName}
           type={txData.type}
           isBuyableChain={isBuyableChain}
+          onClickBlockaidSupport={onClickBlockaidSupport}
```

Comment:
[nit] I think we can move the onClickBlockaidSupport inside of TransactionAlerts. TransactionAlerts seems to only be used for ConfirmTransactionBase

decoupling, and shaving off logic here and 1 less param in TransactionAlerts to be concerned of. Trying to get away from overuse of params. We have a longstanding issue where we overuse parameters and move logic away from its intended purpose. ConfirmTransactionBase seems to be overloaded with logic. 

e.g. some parameters passed to ConfirmTransactionBase
contentComponent - only used by confirm-approve to pass confirm-approve-content
dataComponent - only used by confirm-deploy-contract to pass custom data
isApprovalOrRejection - only used for setApprovalForAll type
subtitle - only used by confirm-token-transaction-base
title- only used by confirm-token-transaction-base and confirm-approve